Index: app/src/main/java/com/neusoft/besterlive/control/activity/LoginActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/neusoft/besterlive/control/activity/LoginActivity.java	(date 1510020102000)
+++ app/src/main/java/com/neusoft/besterlive/control/activity/LoginActivity.java	(revision )
@@ -135,6 +135,7 @@
             public void onSuccess(TIMUserProfile timUserProfile) {
                 //获取用户字段信息成功，将其保存
                 BesterApplication.getApp().saveSelfProfile(timUserProfile);
+
             }
         });
     }
